src/components/Auth/Authenicate.tsx

```
@@ -0,0 +1,250 @@
+              </a>
+            )}
+          </div>
+          <div className="max-w-lg">
+            <h1 className="text-4xl font-black leading-tight tracking-wider text-white lg:text-5xl">
+              {t("care")}
+            </h1>
+            {customDescriptionHtml ? (
+              <div className="py-6">
+                <div
+                  className="max-w-xl text-secondary-400"
+                  dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)